test/models/game_test.rb

```diff
@@ -1,7 +1,33 @@
 require 'test_helper'
 
 class GameTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  def setup
+    @g = Game.create(name: "New Game", white_player_id: 1, black_player_id: 1)
+  end
+
+  test "board populated with pieces" do
```

Comment:
rename this test to more represent what it's testing, in this case that the count of pieces is correct